Sources/Valet/Valet.swift

```diff
@@ -54,6 +54,44 @@ public final class Valet: NSObject {
+    /// Creates a Valet with an explicitly set kSecAttrService.
+    /// See https://github.com/square/Valet/issues/140 for more information.
+    /// - parameter identifier: A non-empty string that uniquely identifies a Valet. Must be unique relative to other Valet identifiers.
+    /// - parameter accessibility: The desired accessibility for the Valet.
+    /// - returns: A Valet that reads/writes keychain elements with the desired accessibility and identifier.
+    public class func valet(withExplicitlySet identifier: Identifier, accessibility: Accessibility) -> Valet {
+        return findOrCreate(explicitlySet: identifier, configuration: .valet(accessibility))
+    }
+
+    /// Creates an iCloud Valet with an explicitly set kSecAttrService.
+    /// See https://github.com/square/Valet/issues/140 for more information.
+    /// - parameter identifier: A non-empty string that uniquely identifies a Valet. Must be unique relative to other Valet identifiers.
+    /// - parameter accessibility: The desired accessibility for the Valet.
+    /// - returns: A Valet (synchronized with iCloud) that reads/writes keychain elements with the desired accessibility and identifier.
+    public class func iCloudValet(withExplicitlySet identifier: Identifier, accessibility: CloudAccessibility) -> Valet {
+        return findOrCreate(explicitlySet: identifier, configuration: .iCloud(accessibility))
+    }
+
+    /// Creates a shared-access-group Valet with an explicitly set kSecAttrService.
+    /// See https://github.com/square/Valet/issues/140 for more information.
+    /// - parameter identifier: A non-empty string that must correspond with the value for keychain-access-groups in your Entitlements file. Must be unique relative to other Valet identifiers.
+    /// - parameter accessibility: The desired accessibility for the Valet.
+    /// - returns: A Valet that reads/writes keychain elements that can be shared across applications written by the same development team.
+    public class func sharedAccessGroupValet(withExplicitlySet identifier: Identifier, accessibility: Accessibility) -> Valet {
+        return findOrCreate(explicitlySet: identifier, configuration: .valet(accessibility), sharedAccessGroup: true)
+    }
+
+    /// Creates an iCloud-shared-access-group Valet with an explicitly set kSecAttrService.
+    /// See https://github.com/square/Valet/issues/140 for more information.
+    /// - parameter identifier: A non-empty string that must correspond with the value for keychain-access-groups in your Entitlements file. Must be unique relative to other Valet identifiers.
+    /// - parameter accessibility: The desired accessibility for the Valet.
+    /// - returns: A Valet (synchronized with iCloud) that reads/writes keychain elements that can be shared across applications written by the same development team.
+    public class func iCloudSharedAccessGroupValet(withExplicitlySet identifier: Identifier, accessibility: CloudAccessibility) -> Valet {
```

Comment:
I think I'm using `class` because of a decision I made years ago where I thought it was nicer? I'll do a separate PR to use `static`, since we're not using subclassing here.